matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/room/model/create/CreateRoomParams.kt

```diff
@@ -61,12 +64,12 @@ open class CreateRoomParams {
      * A list of user IDs to invite to the room.
      * This will tell the server to invite everyone in the list to the newly created room.
      */
-    val invitedUserIds = mutableListOf<String>()
+    var invitedUserIds: MutableList<String> = mutableListOf()
```

Comment:
Was it necessary to change from `val` to  `var`?

Reply:
Yes, otherwise Moshi will not be able to restore this field.
Btw I do not need to explicitly declare the type

Reply:
This is not a data class, when I originally tried by keeping the `val`, some fields were not persisted, I thought that it was for another reason but changing to `var` fixed the issue. I suppose that as these are not constructor fields, Moshi cannot do the same trick 😞

Reply:
> This is not a data class

My bad, OK then!